app/mailers/activity_mailer.rb

```diff
@@ -23,7 +24,7 @@ def mail(options)
     template = options.delete(:template)
     if template
       prepend_view_path(File.dirname(template))
-      options[:template_path] = ''
+      options[:template_path] = File.dirname(template)
```

Comment:
then why keep prepend_view_path? What is the spec that breaks with prepend_view_path?